### PR TITLE
RH6: hv_netvsc: split sub-channel setup into async and sync

### PR DESCRIPTION
**Pull request by johnsongeorge-w**
Backported from RH7: SHA: 673f5302723eacc3b9172e6ab25927bc59b75269

### CODE REVIEW

**Comment by johnsongeorge-w** [comment]
@santoshx @prashantmhetar Please review

**Comment by johnsongeorge-w** [comment]
@santoshx Fixed the space.

File: hv-rhel6.x/hv/netvsc_drv.c [hunk 1]
```diff
@@ -2435,6 +2448,9 @@ static int netvsc_probe(struct hv_device *dev,
 	}
 	memcpy(net->dev_addr, device_info.mac_adr, ETH_ALEN);
 
+	if (nvdev->num_chn > 1)
+                schedule_work(&nvdev->subchan_work);
```
Comment (santoshx) [inline hv-rhel6.x/hv/netvsc_drv.c]: spaces instead of tab

**Comment by johnsongeorge-w** [comment]
@santoshx Done

**Pull request merged**